src/http/api/v1/matches.ts

```diff
@@ -111,5 +113,61 @@ export const createHandler = (config: HttpApiConfig): express.Router => {
     res.send(responseBody);
   }
 
+  router.post("/chains/:chainId/rounds/:roundId/estimate", async (req, res) => {
+    await estimateMatchesHandler(req, res, 200);
+  });
+
+  async function estimateMatchesHandler(
+    req: Request,
+    res: Response,
+    okStatusCode: number
+  ) {
+    const chainId = Number(req.params.chainId);
+    const roundId = req.params.roundId;
+    const potentialVotes = potentialVotesSchema
+      .parse((req.body as { potentialVotes: PotentialVotes }).potentialVotes)
```

Comment:
The cast here shouldn't be necessary as the type is taken care of by zod.

Reply:
True - missed this. fixed.